src/features/import/utils/prepareImportOperations.ts

```diff
@@ -52,22 +86,26 @@ export default function prepareImportOperations(
               if (!personImportOps[rowIndex].tags) {
                 personImportOps[rowIndex].tags = [];
               }
-              personImportOps[rowIndex].tags = [
-                ...new Set(
-                  personImportOps[rowIndex].tags?.concat(mappedColumn.tagIds)
-                ),
+              const allTags = personImportOps[rowIndex].tags?.concat(
+                mappedColumn.tags.map((t) => ({ tag_id: t.id }))
```

Comment:
Yes, this was done consciously to match the current API so that it was possible to test it. Have a note about changing this as soon as the API changes, but now that we know that an import is possible, I guess it works to make the change on the frontend even though it's not yet depoloyed on the backend :D